src/bitmap/inherent.rs

```diff
@@ -43,6 +43,41 @@ impl RoaringBitmap {
         container.insert(index)
     }
 
+    /// Adds a value to the set.
+    /// The value **must** be strictly bigger than the maximum value in the set.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use roaring::RoaringBitmap;
+    ///
+    /// let mut rb = RoaringBitmap::new();
+    /// rb.push(1);
+    /// rb.push(3);
+    /// rb.push(5);
+    ///
+    /// assert_eq!(rb.iter().collect::<Vec<u32>>(), vec![1, 3, 5]);
+    /// ```
+    pub fn push(&mut self, value: u32) -> bool {
```

Comment:
What does the boolean value means? If this method is only able to insert at the end an integer that do not already exist it should always return `true`, or does it return `false` when the operation is not a success?

You could maybe add a note about the returned value? :)

Reply:
Sure! It's like the in the insert method, basically it return false if it's a duplicate of the last value. Sorry for the mess, I'll have dinner and this evening or in the morning I'll extend the documentation.

Reply:
After sleeping on it, I think that, as you pointed out, the bool return might be confusing so I removed it.